# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/public/mew/Makefile.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	Wed Jul 18 01:20:40 2001
+++ ./Makefile	Wed Oct 31 11:43:09 2001
@@ -19,30 +19,30 @@
 ## EDIT THE FOLLOWINGS
 ##
 
-prefix=/usr/local
+prefix=/usr
 infodir=$(prefix)/info
 
 ##
 ## Compile engines
 ##
 
-EMACS = emacs
-#EMACS = xemacs
+#EMACS = emacs
+EMACS = xemacs
 #EMACS = mule
 
 ##
 ## A directory where mew*.el[c] will be installed.
 ##
 
-elispdir = $(prefix)/share/emacs/site-lisp/mew
-#elispdir  = $(prefix)/lib/$(EMACS)/site-lisp
+#elispdir = $(prefix)/share/emacs/site-lisp/mew
+elispdir  = $(prefix)/lib/$(EMACS)/xemacs-packages/lisp/mew
 #elispdir = $(prefix)/lib/emacs
 
 ##
 ## A directory where etc file will be installed.
 ##
 
-etcdir = $(prefix)/lib/$(EMACS)/etc/Mew
+etcdir = $(prefix)/lib/$(EMACS)/xemacs-packages/etc/mew
 
 ################################################################
 ##
